src/include/daos_fs_sys.h

```diff
@@ -180,15 +180,17 @@ dfs_sys_access(dfs_sys_t *dfs_sys, const char *path, int mask, int flags);
 /**
  * Change permission access bits. Symlinks are dereferenced.
  *
- * \param[in]	dfs_sys Pointer to the mounted file system.
- * \param[in]	path	Link path of object.
- * \param[in]	mode	New permission access modes. For now, we don't support
- *			the sticky bit, setuid, and setgid.
+ * \param[in]	dfs_sys 		Pointer to the mounted file system.
+ * \param[in]	path			Link path of object.
+ * \param[in]	mode			New permission access modes. For now,
+ * 					we don't support the sticky bit, setuid,
+ * 					and setgid.
+ * \param[in]	ignore_notsup_mod 	Ignore unsupported file mode bits.
```

Comment:
Likewise here... We're making a public API change, so it would probably be best to generalize, as I suspect that callers won't really care about having fine-grained control over which unsupported features should be ignored.

Reply:
Done.